ext/bindgen_helper.hpp

```diff
@@ -39,11 +39,12 @@
 
 #ifndef BINDGEN_HELPER_HPP
 #define BINDGEN_HELPER_HPP
+#define GC_NEW_ABORTS_ON_OOM
```

Comment:
I don't even remember anymore why this file is mirrored from Bindgen.  In any case, could you mirror the changes over?  That define sounds like a good idea!